tests/lib/rules/jsx-no-literals.js

```diff
@@ -368,20 +437,6 @@ ruleTester.run('jsx-no-literals', rule, {
         {message: stringsMessage('`foo`')},
         {message: stringsMessage('\'bar\'')}
       ]
-    }, {
-      code: '<Foo bar={`foo` + `bar`} />',
-      options: [{noStrings: true}],
```

Comment:
why do these test cases need to be deleted? if it's behind an option, they should remain passing.

Reply:
these tests seemed duplicate to me, I didn't notice the bar was string literal in first and foo was string in another. will add them again.

Reply:
Added tests.